app/controllers/contributors_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ContributorsController < ApplicationController
+  before_action :check_if_feature_flag_is_on
+
+  def new
+    @contributor = Contributor.new
+  end
+
+  def create
+    @contributor = Contributor.new(contributor_params)
+    @suggested_councillor = SuggestedCouncillor.find(params[:contributor][:suggested_councillor_id])
```

Comment:
What happens if the councillor can't be found?

If you use a bang method here (i.e. `.find!`) then it would raise a 404 if it couldn't find the councillor. This is usually really handy to use in situations like this however it's probably not the behaviour we want.